src/WattleScript.Interpreter/Execution/VM/Processor/Processor_InstructionLoop.cs

```diff
@@ -1224,15 +1224,19 @@ private int ExecBinaryOp(int instructionPtr, string metaMethodName)
 		
 		private int ExecAddStr(int instructionPtr)
 		{
-			if (m_ValueStack.Peek().TryCastToNumber(out var rn) && 
-			    m_ValueStack.Peek(1).TryCastToNumber(out var ln))
+			DynValue dvA = m_ValueStack.Peek();
```

Comment:
This should be `ref DynValue dvA` if you want to store the reference as a local. Otherwise this is introducing an unnecessary copy

Reply:
Added in [9e37993](https://github.com/WattleScript/wattlescript/pull/36/commits/9e37993f040ba71f1b9866dc8a08ea777e79d53a)